src/components/sections/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { memo } from 'react'
+import type { BreadcrumbProps } from 'src/components/ui/Breadcrumb'
+import UIBreadcrumb from 'src/components/ui/Breadcrumb'
+
+interface BreadcrumbWrapperProps
+  extends Partial<Pick<BreadcrumbProps, 'breadcrumbList'>> {
+  name: string
+}
+
+function Breadcrumb({ breadcrumbList, name }: BreadcrumbWrapperProps) {
+  const fallback = [{ item: '/', name, position: 1 }]
```

Comment:
please move this to the global scope so we don't re-create this object at every render

Reply:
This array needs to be recreated due to the property name, I thought the same, but the name comes from props.